src/lib/LocalMedia.ts

```diff
@@ -44,9 +44,19 @@ const TypedLocalMediaEventTarget = EventTarget as { new (): LocalMediaEventTarge
 export default class LocalMedia extends TypedLocalMediaEventTarget {
     private _constraints: MediaStreamConstraints | null = null;
     public _rtcManagers: RtcManager[];
+
     public stream: MediaStream;
     public screenshareStream?: MediaStream;
 
+    // Camera state
+    private _cameraEnabled = false;
```

Comment:
The tests seems to be fail because the cameras are off initially now. Not sure if because of this line or not, but opening the test app and joining a room I seem my local participant's camera off until I explicitly enable it.

(At least locally, but I assume it's the same on CI)

<img width="1505" alt="Screenshot 2023-10-24 at 14 31 12" src="https://github.com/whereby/browser-sdk/assets/1159931/b6d3d500-fdf2-4fde-b8b3-784adcb58602">

Reply:
I know why these are failing, fix incoming

Reply:
It is in the case of explicitly passed localStream, then we basically end up with the incorrect local state for cam / mic enabled

Reply:
Should be fixed now